packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Models/Serialization/SerializationInterfaces.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Generator.CSharp
+{
+    public abstract class SerializationInterfaces : IEnumerable<CSharpType>
+    {
+        protected SerializationInterfaces(TypeProvider typeProvider, bool hasJson, bool hasXml) { }
```

Comment:
If this is a publicly exposed API, it seems like it may not be the most extensible if we need to keep adding new parameters for new formats.

Reply:
Resolving as this class was refactored and no longer exists.